server/router/driver.ts

```diff
@@ -159,7 +159,11 @@ router.put('/:id', validateUser('Driver'), (req, res) => {
     params: { id },
     body,
   } = req;
-  db.update(res, Driver, { id }, body, tableName);
+  if (id === res.locals.user.id) {
```

Comment:
we want admins to be able to edit all users, so allow edits if `userType === UserType.ADMIN` too